test/create_es_test.go

```diff
@@ -0,0 +1,48 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformCreateES(t *testing.T) {
+	// This creates this function with the possibility of being run in parallel with other functions
+	// that start with this same t.Parallel() call.
+	t.Parallel()
+
+	namePrefix := "terratest-es"
```

Comment:
If tests are running in parallel, will resources conflict?

Should we do something like add a UUID to this value?